api-test/src/main/resources/config/testCaseInterDependency.json

```
@@ -1,4 +1,6 @@
 {
+    "TC_PMS_DeleteBioExtractorConfiguration_02": ["TC_PMS_CreateBioExtractorConfiguration_32"],
```

Biome (2.5.0)

[error] The key TC_PMS_DeleteBioExtractorConfiguration_02 was already declared.

(lint/suspicious/noDuplicateObjectKeys)